### PR TITLE
[Rule Tuning] Windows DR Tuning - 14

### PR DESCRIPTION
**Pull request by w0rk3r**
## Issues

Part of #3186 

## Summary

Tunes the following rules:

* Component Object Model Hijacking
* Potential Persistence via Time Provider Modification
* Web Shell Detection: Script Process Child of Common Web Processes
* Expired or Revoked Driver Loaded
* Suspicious PrintSpooler Service Executable File Creation

### CODE REVIEW

File: rules/windows/persistence_suspicious_com_hijack_registry.toml [hunk 1]
```diff
@@ -13,7 +13,7 @@ Identifies Component Object Model (COM) hijacking via registry modification. Adv
 executing malicious content triggered by hijacked references to COM objects.
 """
 from = "now-9m"
-index = ["logs-endpoint.events.*", "endgame-*"]
+index = ["logs-endpoint.events.*"]
```
Comment (w0rk3r) [inline rules/windows/persistence_suspicious_com_hijack_registry.toml:16]: I removed Endgame support to check signer info; Endgame doesn't have this data outside process events.

File: rules/windows/persistence_suspicious_com_hijack_registry.toml [hunk 2]
```diff
@@ -84,39 +84,110 @@ registry where host.os.type == "windows" and
...
+              /* https://strontic.github.io/xcyclopedia/library/clsid_4ED3A719-CEA8-4BD9-910D-E252F997AFC2.html */
+              "{4ED3A719-CEA8-4BD9-910D-E252F997AFC2}",
+
+              /* https://strontic.github.io/xcyclopedia/library/clsid_A56A841F-E974-45C1-8001-7E3F8A085917.html */
+              "{A56A841F-E974-45C1-8001-7E3F8A085917}",
+
+              /* https://strontic.github.io/xcyclopedia/library/clsid_BFEC0C93-0B7D-4F2C-B09C-AFFFC4BDAE78.html */
+              "{BFEC0C93-0B7D-4F2C-B09C-AFFFC4BDAE78}",
+              "%SystemRoot%\\system32\\shdocvw.dll"
+            )
```
Comment (w0rk3r) [inline rules/windows/persistence_suspicious_com_hijack_registry.toml:122]: A second 👀 to confirm if we can exclude those would be nice.

Cc @Samirbous

File: rules/windows/privilege_escalation_expired_driver_loaded.toml [hunk 3]
```diff
@@ -20,9 +20,9 @@ name = "Expired or Revoked Driver Loaded"
 references = [
   "https://docs.microsoft.com/en-us/previous-versions/windows/hardware/design/dn653559(v=vs.85)?redirectedfrom=MSDN"
 ]
-risk_score = 21
+risk_score = 47
 rule_id = "d12bac54-ab2a-4159-933f-d7bcefa7b61d"
-severity = "low"
+severity = "medium"
```
Comment (w0rk3r) [inline rules/windows/privilege_escalation_expired_driver_loaded.toml:25]: Low volume, bumping severity

File: rules/windows/privilege_escalation_printspooler_service_suspicious_file.toml [hunk 4]
```diff
@@ -41,11 +41,16 @@ query = '''
...
-           "?:\\WINDOWS\\system32\\PrintConfig.dll")
+  not file.path : (
+    "?:\\WINDOWS\\SysWOW64\\PrintConfig.dll",
+    "?:\\WINDOWS\\system32\\x5lrs.dll",
+    "?:\\WINDOWS\\sysWOW64\\x5lrs.dll",
+    "?:\\WINDOWS\\system32\\PrintConfig.dll",
+    "?:\\WINDOWS\\system32\\spool\\DRIVERS\\x64\\*.dll",
+    "?:\\WINDOWS\\system32\\spool\\DRIVERS\\W32X86\\*.dll",
+    "?:\\WINDOWS\\system32\\spool\\PRTPROCS\\x64\\*.dll",
+    "?:\\WINDOWS\\system32\\spool\\{????????-????-????-????-????????????}\\*.dll"
```
Comment (w0rk3r) [inline rules/windows/privilege_escalation_printspooler_service_suspicious_file.toml:52]: Super high volume (>900k events last 90d), ?s cover the GUID format

**Review by Aegrah — APPROVED** [review]
Nice work. Glad to see the component object model hijacking rule receive some good tuning!

File: rules/windows/persistence_suspicious_com_hijack_registry.toml (hunk 1 above)
Comment (terrancedejesus) [reply rules/windows/persistence_suspicious_com_hijack_registry.toml:16]: Good call

File: rules/windows/persistence_suspicious_com_hijack_registry.toml [hunk 5]
```diff
@@ -84,39 +84,110 @@ registry where host.os.type == "windows" and
   user.domain != "NT AUTHORITY" and
   (
     (
-      registry.path : ("HK*\\InprocServer32\\", "\\REGISTRY\\*\\InprocServer32\\") and
-      registry.data.strings: ("scrobj.dll", "C:\\*\\scrobj.dll") and
+      registry.path : "HK*\\InprocServer32\\" and
+      registry.data.strings: ("scrobj.dll", "?:\\*\\scrobj.dll") and
```
Comment (terrancedejesus) [inline rules/windows/persistence_suspicious_com_hijack_registry.toml:88]: Almost feel like we should audit all Windows rules and check for this.

File: rules/windows/persistence_webshell_detection.toml [hunk 6]
```diff
@@ -4,8 +4,7 @@ integration = ["endpoint", "windows"]
 maturity = "production"
 min_stack_comments = "New fields added: required_fields, related_integrations, setup"
 min_stack_version = "8.3.0"
-updated_date = "2023/10/23"
-
+updated_date = "2024/01/09"
```
Comment (w0rk3r) [inline rules/windows/persistence_webshell_detection.toml:7]:
```suggestion
updated_date = "2024/01/09"

```

**Review by terrancedejesus — APPROVED** [review]
Great stuff!

**Pull request merged**